assets/src/edit-story/app/story/actions/useSavePost.js

```diff
@@ -1,13 +1,34 @@
 /**
  * WordPress dependencies
  */
-import { useCallback } from '@wordpress/element';
+import { useCallback, renderToString } from '@wordpress/element';
 import { addQueryArgs } from '@wordpress/url';
 
 /**
  * Internal dependencies
  */
 import { useAPI } from '../../api';
+import { getDefinitionForType } from '../../../elements';
+
+const getStoryMarkupFromPages = ( pages ) => {
+	// @todo look into: We should rather use renderToStaticMarkup here, however, it doesn't seem to be exposed via @wordpress/element.
+	const markup = pages.map( ( page ) => {
+		const { id } = page;
+		return renderToString(
+			<amp-story-page id={ id }>
+				<amp-story-grid-layer template="vertical">
+					{ page.elements.map( ( { type, ...rest } ) => {
+						const { id: elId } = rest;
+						// eslint-disable-next-line @wordpress/no-unused-vars-before-return
+						const { Save } = getDefinitionForType( type );
+						return <Save key={ 'element-' + elId } { ...rest } />;
+					} ) }
+				</amp-story-grid-layer>
+			</amp-story-page>,
```

Comment:
Is this comma doing anything?

Reply:
It's the mandatory trailing comma.

Reply:
@miina This comma now seems to be saved as part of post_content.

Reply:
Hmph, okay, removed and added eslint ignore: https://github.com/ampproject/amp-wp/pull/3999